src/openrct2/actions/WaterLowerAction.h

```diff
@@ -11,6 +11,8 @@
 
 #include "GameAction.h"
 
+struct MapRange;
```

Comment:
This shouldn't be here. MapRange is included from GameAction which includes it from GameActionResult which includes Location.hpp. Even if it didn't this still doesn't make sense as you cant forward declare a value parameter like we have with MapRange.

Reply:
Fair enough, it was only an attempt to get the Debian CI to compile.